crates/indexer/src/geyser/accounts/metadata.rs

```diff
@@ -2,16 +2,28 @@ use indexer_core::{
     db::{
         custom_types::TokenStandardEnum,
         insert_into,
-        models::{Metadata, MetadataCollectionKey, MetadataCreator},
-        tables::{metadata_collection_keys, metadata_creators, metadatas},
+        models::{FeedEventWallet, Metadata, MetadataCollectionKey, MetadataCreator, MintEvent},
+        select,
+        tables::{
+            feed_event_wallets, feed_events, metadata_collection_keys, metadata_creators,
+            metadatas, mint_events,
+        },
     },
+    prelude::*,
     pubkeys::find_edition,
+    uuid::Uuid,
 };
 use mpl_token_metadata::state::{Collection, Metadata as MetadataAccount, TokenStandard};
 
 use super::Client;
 use crate::prelude::*;
 
+#[derive(Clone, Copy)]
+enum FeedEventId {
+    Existing,
+    Inserted(Uuid),
+}
```

Comment:
This (and the previous `Either` version) is isomorphic to `Option<Uuid>`.  Is there any reason we don't just do that?

Reply:
word. switched to Option.